.github/workflows/cd.yml

```diff
@@ -13,12 +13,12 @@ jobs:
 
   call-publish-docker:
     name: Build and Publish Docker image
-    uses: eclipse-pass/playground/.github/workflows/html-app/publish.yml@main
+    uses: eclipse-pass/playground/.github/workflows/html-app-publish.yml@main
```

Comment:
Subduers caused a problem? Or just easier within one dir?

Reply:
GH Actions doesn't allow a workflow to be run from a subfolder. There is a different mechanism that allows subfolders to be used but the intent of these files is that they will be copied into the `workflows` folder of a different repo so I thought it was best to use a naming convention instead of having two different file formats.